playbooks/openshift-node/private/restart.yml

```diff
@@ -16,7 +16,9 @@
     until: not (l_docker_restart_docker_in_node_result is failed)
     retries: 3
     delay: 30
-    when: openshift_node_restart_docker_required | default(True)
+    when: 
+      - openshift_node_restart_docker_required | default(True)
+      - not openshift_use_crio_only | bool
```

Comment:
@mtnbikenc do you think we should keep this check?

Reply:
It still seems reasonable to keep the crio check, because if a restart is called on nodes without docker installed it would fail.